packages/core/pvm-host-calls/host-calls-manager.ts

```diff
@@ -54,7 +54,7 @@ export class HostCallsManager {
         return `r${idx}=${value} (0x${value.toString(16)})`;
       })
       .join(", ");
-    logger.insane`[${currentServiceId}] ${context} ${name}${requested}.  Gas: ${gas}. Regs: ${registerValues}.`;
+    logger.trace`[${currentServiceId}] ${context} ${name}${requested}.  Gas: ${gas}. Regs: ${registerValues}.`;
```

Comment:
```suggestion
    logger.insane`[${currentServiceId}] ${context} ${name}${requested}.  Gas: ${gas}. Regs: ${registerValues}.`;
```
Please revert, that should be `insane` log level.

If you want to display this please just set `JAM_LOG=insane`, or more selectively `JAM_LOG=trace,host-calls-pvm=insane`